pkg/operator/sync.go

```diff
@@ -126,6 +142,74 @@ func (optr *Operator) syncTerminationHandler(config *OperatorConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncWebhookConfiguration() error {
+	if err := optr.syncValidatingWebhook(); err != nil {
+		return err
+	}
+
+	return optr.syncMutatingWebhook()
+}
+
+func (optr *Operator) syncValidatingWebhook() error {
+	client := optr.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations()
+	requiredWebhookConfiguration := mapiv1.NewValidatingWebhookConfiguration()
```

Comment:
this is confusing, why the inconsistency between naming `requiredWebhookConfiguration` vs `currentHook`? is WebhookConfiguration different than a Hook?

We only need to have two different names across the whole code